### PR TITLE
Changes for video-calls

### PR DESCRIPTION
**Pull request by sergiohs84**

### CODE REVIEW

**Comment by polmr** [comment]
trigger compilation

File: src/megaapi_impl.cpp [hunk 1]
```diff
@@ -10536,8 +10536,8 @@ void MegaApiImpl::notify_confirmation(const char *email)
 
 void MegaApiImpl::notify_disconnect()
 {
-    MegaEventPrivate *event = new MegaEventPrivate(MegaEvent::EVENT_DISCONNECT);
-    fireOnEvent(event);
+    //MegaEventPrivate *event = new MegaEventPrivate(MegaEvent::EVENT_DISCONNECT);
+    //fireOnEvent(event);
```
Comment (sergiohs84) [inline src/megaapi_impl.cpp]: Once @alber2510 changes MEGAchat to not disconnect on this event if there's an on-going call, I think this callback is suitable and should be kept here.

File: src/megaclient.cpp [hunk 2]
```diff
@@ -9871,6 +9871,12 @@ void MegaClient::queueread(handle h, bool p, SymmCipher* key, int64_t ctriv, m_o
     else
     {
         it->second->enqueue(offset, count, reqtag, appdata);
+        if (overquotauntil && overquotauntil > Waiter::ds)
```
Comment (sergiohs84) [inline src/megaclient.cpp]: @javiserrano are you aware of this change? it's not in `develop` yet, so I wonder how it ended up here in this branch

Comment (javiserrano) [reply src/megaclient.cpp]: Yes, it's to fix the management of the overquota state on streaming transfers.
The problem was reported by Android developers and the fix has been already confirmed to be working fine.

**Pull request merged**